grim.go

```diff
@@ -146,6 +151,17 @@ func (i *Instance) BuildNextInGrimQueue(logger *log.Logger) error {
 	return nil
 }
 
+func repoIsAlive(hook *hookEvent) bool {
+	url := hook.URL
+	if len(url) >= 12 {
+		lastCommitNum := url[len(url)-12 : len(url)]
```

Comment:
url[len(url)-12: len(url)] should be same as url[len(url)-12: ], just nitpicking but looks good.